### ISSUE TITLE
Add an $IsAzureFunctions

### ISSUE THREAD

**Issue by tylerl0706** [Azure/azure-functions-powershell-worker#81]
Similar to `$IsWindows`, `$IsMacOS`, `$IsLinux`

Just a helper global variable that says whether a script is running in Azure Functions

cc/ @joeyaiello @SteveL-MSFT

**Comment by JustinGrote** [comment]
While there's other ways to check for this in general, I agree it's low
impact and low effort, no valid reason not to have it other than backlog
(and unlikely but possible variable conflicts for those new to the
platform)

**Comment by tylerl0706** [comment]
Alternatively, if Azure Functions already exposes an Environment variable, we could possibly leave it at that.

@asavaritayal @pragnagopa and idea if this is a thing?

**Comment by dfinke** [comment]
Was thinking about how to get that the other day. I want to test my PS code locally but emit the results to `$res` if I'm running as an Azure Function.

**Comment by TylerLeonhardt** [comment]
Good point @dfinke something like this could enable users to write scripts that would work locally and in Azure Functions.

**Comment by joeyaiello** [comment]
Let's talk to the AzF team about adding an environment variable to the host that tells you you're in an Azure Function.

**Comment by TylerLeonhardt** [comment]
Today they rely on:

`WEBSITE_INSTANCE_ID`

It would probably be nice to have a more customer friendly env var.

**Comment by anirudhgarg** [comment]
There are a number of this including WEBSITE_INSTANCE_ID.
